### PR TITLE
PMM-11268: Verify tooltips work for DBaaS page

### PR DESCRIPTION
**Pull request by nazarovvi**

### CODE REVIEW

**Comment by nazarovvi** [comment]
image

File: tests/pages/adminPage.js [hunk 1]
```diff
@@ -175,4 +177,35 @@ module.exports = {
...
+    const tooltipIcon = tooltipObj.iconLocator;
+    const tooltipText = tooltipObj.tooltipText.as('Tooltip text');
+
+    I.waitForVisible(tooltipIcon, 5);
+    I.moveCursorTo(tooltipIcon);
+    I.scrollTo(tooltipText);
+    I.waitForVisible(tooltipText, 5);
+    I.seeTextEquals(tooltipObj.text, tooltipText);
+    /* there are tooltip without "Read more" link */
+    if (tooltipObj.link) {
```
Comment (vyarosh) [inline tests/pages/adminPage.js:200]: this will cause "cannot ready property of unrefined" error. Please add `Object.hasOwn(tooltipObj, 'link')` first.

Comment (nazarovvi) [reply tests/pages/adminPage.js:200]: image
I don't think that's the case

**Review by puneet0191 — APPROVED** [review]
@nazarovvi please add log of execution on an actual pipeline, probably settings would be a good place for these,if not dbaas

File: tests/pages/adminPage.js [hunk 2]
```diff
@@ -175,4 +177,35 @@ module.exports = {
...
+      const tooltipReadMoreLink = tooltipObj.tooltipReadMoreLink.as(`Tooltip "Read more" link for ${tooltipObj.iconLocator}`);
+
+      I.scrollTo(tooltipReadMoreLink);
+      I.seeAttributesOnElements(tooltipReadMoreLink, { href: tooltipObj.link });
+      const readMoreLink = (await I.grabAttributeFrom(tooltipReadMoreLink, 'href'));
+      const response = await I.sendGetRequest('https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types1233333333.html');
```
Comment (beatahandzelova) [inline tests/pages/adminPage.js:206]: I think you forgot to replace the "https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types1233333333.html" here?

Comment (nazarovvi) [reply tests/pages/adminPage.js]: `const response = await I.sendGetRequest(readMoreLink);`
replaced with readMoreLink

**Comment by nazarovvi** [comment]
image
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-e2e-tests/detail/pmm2-dbaas-e2e-tests/2112/tests

File: tests/DbaaS/verifyDBaaS_test.js [hunk 3]
```diff
@@ -344,3 +345,34 @@ Failed to register pmm-agent on PMM Server: Post "https://https:%2F%2F1.2.3.4/v1
   },
 );
 
+Scenario('@PMM-T1512 Verify tooltips work properly for DBaaS page @dbaas',
+  async ({
+    I, dbaasPage, adminPage, settingsAPI,
+  }) => {
+    let turnOffDbaas;
+
+    if (!(await settingsAPI.getSettings('dbaas_enabled'))) {
```
Comment (beatahandzelova) [inline tests/DbaaS/verifyDBaaS_test.js:354]: This check can be removed, dbaas tests run with -e ENABLE_DBAAS=1 so situation where dbaas is disabled and needs to be enabled cannot happen.

**Review by puneet0191 — APPROVED** [review]

File: tests/DbaaS/verifyDBaaS_test.js (hunk 3 above)
Comment (nazarovvi) [reply tests/DbaaS/verifyDBaaS_test.js]: Removed. @beatahandzelova could you check again?

**Pull request merged**